extensions-contrib/cloudfiles-extensions/src/main/java/io/druid/storage/cloudfiles/CloudFilesDataSegmentPusher.java

```diff
@@ -146,4 +148,19 @@ public DataSegment call() throws Exception
       }
     }
   }
+
+  @Override
+  public Map<String, Object> makeLoadSpec(URI uri)
```

Comment:
minor nit : for each pusher the makeLoadSpec logic seems duplicated with the push method, this can be extracted to a common method.